### DISCUSSION TITLE
Selecting multiple notes instead of editing another one

### DISCUSSION THREAD

**Discussion by fixje** [standardnotes/forum#2747] [General]
I came across some unintended and annoying behavior with Standard Notes in my browser (recent Chrome on Linux).

I haven't found an exact path for reproducing, but here is what happens:
- I edit some note (it's a pinned note usually)
- I want to edit a different note and select it in the list of notes on the left using the mouse. It doesn't matter whether it's pinned or not, or, if I use the search bar first or not.
- Instead of opening the note in the editor, the editor goes empty and shows "2 notes selected"

I would expect this to happen when I do something like holding `Ctrl` while selecting the other note, but that's not the case. I always need to click on the note I had previously opened to de-select it, before I can proceed.

I observed this many times now, and it usually happens after not using the UI for a couple of minutes (Chrome or the tab went to background). If it helps: I use the Markdown Pro Editor v1.3.13

**Comment by JaspalSuri** [comment]
Hi @fixje, sorry to hear that. Thank you for those details and the list of reproduction steps. I'm afraid that I wasn't able to reproduce the issue.

If you have a detachable keyboard, can you still reproduce the issue by temporarily disconnecting the keyboard? Use an on-screen keyboard that the system provides, disconnect the physical keyboard, then try to reproduce the bug.

**Comment by fixje** [comment]
Hi @JaspalSuri, I don't have an external keyboard at the moment, because I am working with a laptop.

Your approach with the on-screen keyboard sounds interesting. Could you explain what is the expected cause of the problem in this context?

**Comment by JaspalSuri** [comment]
Hi @fixje, thank you for letting me know. Can you attach an external keyboard to the laptop (if you're comfortable with doing so)? 

>Your approach with the on-screen keyboard sounds interesting. Could you explain what is the expected cause of the problem in this context?

In this case it appears to be a hardware related issue, but that would make sense if you were experiencing it in other apps. A colleague of mine and I weren't able to reproduce the issue so far (on Linux and other platforms).

**Comment by adamhotep** [comment]
I also experience this issue (Firefox 102.0 on Debian, app.standardnotes.org). It happens quite frequently, but I have not been able to come up with a method to reliably reproduce it. I'm not using an on-screen keyboard nor am I using any keyboard shortcuts (a la standardnotes/forum#1844). When it happens, I often struggle to get it back to a single note other than the original. 

I'd personally just prefer to disable multi-selection. I've never wanted to do it, though I understand it has its uses (like mass-deletion). I like the way iOS handles it, as a dedicated mode that doesn't get in the way of other workflows. Putting multi-select into the context menu should work.

**Comment by guido4000** [comment]
Why has this been converted into a discussion? It is a super annoying bug present in multiple browsers including the current Chrome.

**Comment by atspaeth** [comment]
I have the same issue on the app version on Ubuntu 23.04 on both of my computers (one a laptop running xorg and one a desktop running wayland). Doesn't happen on the web version on Firefox (which in my quick tests seems to have a _different_ multi-select issue related to control-tabbing, but I don't use it anyway).

For me it's 100% repeatable. If at any point during editing any note I press shift, whether I actually type an uppercase letter or not, then the next time I try to change notes, it selects the entire range between the current note and the one I clicked. The only way I've found to cancel it is to discharge the multi-select state by clicking the same note I'm already editing before trying to switch, or reset it by changing the view to a different tag.
